scripts/install-dev.sh

```diff
@@ -349,9 +349,12 @@ install_deno_vscode_extension() {
 }
 
 feedback_message() {
-        if [ $(command -v $SLACK_CLI_NAME) ]; then
+        if [ $? -eq 0 ] && [ $(command -v $SLACK_CLI_NAME) ]; then
```

Comment:
📣 This function is now called on errors too - we check the status before it's called to change outputs here!